f/fftw/manifest.x86_64.jsonc

```diff
@@ -0,0 +1,74 @@
+/** Human readable report. This is not consumed by boulder */
+{
+	"manifest-version": "0.2",
+	"packages": {
+		"fftw": {
+			"depends": [
+				"interpreter(/usr/lib/ld-linux-x86-64.so.2(x86_64))",
+				"soname(libc.so.6(x86_64))",
+				"soname(libm.so.6(x86_64))"
+			],
+			"files": [
+				"/usr/bin/fftw-wisdom",
+				"/usr/bin/fftw-wisdom-to-conf",
+				"/usr/bin/fftwf-wisdom",
+				"/usr/lib/libfftw3.so.3",
+				"/usr/lib/libfftw3.so.3.6.10",
+				"/usr/lib/libfftw3_threads.so.3",
+				"/usr/lib/libfftw3_threads.so.3.6.10",
+				"/usr/lib/libfftw3f.so.3",
+				"/usr/lib/libfftw3f.so.3.6.10",
+				"/usr/lib/libfftw3f_threads.so.3",
+				"/usr/lib/libfftw3f_threads.so.3.6.10",
```

Comment:
These libs should be subpackaged into a `-libs` package

Reply:
could you explain why? what happens to just fftw would it then depend on the libs folder or should the application in this case(easyeffects) pull them in when doing pkgconfig(fftwf) for example?

Reply:
also what do you mean these libs? all of lib? or just ones with libfftw* and don't do that for libfftw.so* files?